packages/shared/lib/validateAddress.ts

```diff
@@ -104,6 +104,9 @@ export class AddressValidator {
         return this.validateTERRA(address);
       case 'TON':
       case 'TON_TEST':
+      case 'USDT_TON':
+      case 'DOGS_TON':
+      case 'NOTCOIN_TON':
```

Comment:
See if we can generalize to just "Jetton" or work by protocol instead of asset ID, if we use assetId this switch-case will be massive